src/Bundle/Extractor/Property.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Zenstruck\Foundry\Bundle\Extractor;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+use ReflectionClass;
+
+class Property
+{
+    /**
+     * @var array
+     */
+    private $properties = [];
+
+    /**
+     * @var array
+     */
+    private $defaultProperties = [];
+
+    /**
+     * @var EntityManagerInterface
+     */
+    private $em;
+
+    public function __construct(EntityManagerInterface $em)
+    {
+        $this->em = $em;
+    }
+
+    /**
+     * @TODO Support for defaults is missing
```

Comment:
What are you meaning by "defaults" here?

Reply:
i removed defaults. This was from early step where i forgot thats a default will be always available on instantiates classes. Defaults meaning: `$foo = 'bar` as examble.